frontend/src/concepts/k8s/utils.ts

```diff
@@ -4,7 +4,7 @@ import { genRandomChars } from '~/utilities/string';
 
 export const PreInstalledName = 'Pre-installed';
 
-export const ownedByDSC = (resource: K8sResourceCommon): boolean =>
+export const isOwnedByDSC = (resource: K8sResourceCommon): boolean =>
```

Comment:
When you rebase this on top of the fix Griffin did... please rename this method to something like `isOOTB` as it's more important to know that this is "an out of the box" item, and not something owned by the operator/dsc (since it's not what we are after here)

Reply:
yeah sure, Thanks.